scylla-cdc/src/consumer.rs

```diff
@@ -58,6 +74,8 @@ const TTL_NAME: &str = "cdc$ttl";
 const IS_DELETED_PREFIX: &str = "cdc$deleted_";
 const ARE_ELEMENTS_DELETED_PREFIX: &str = "cdc$deleted_elements_";
 
+/// An internal structure used to map names of columns in the CDC
+/// to their indices in an internal vector.
```

Comment:
In what sense it is internal? It is a part of the public API.

Reply:
To be honest, it shouldn't be public. But we can't create the replication tests without it being public (we don't use the whole library there and we can't create `CDCRowSchema` if it's not public). I will just remove the adjective.